src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -1115,18 +1118,18 @@ void eventer_ssl_init() {
   SSL_library_init();
   OpenSSL_add_all_ciphers();
 
-  if (!dh512_file || strcmp(dh512_file, "")) {
+  if (!dh1024_file || strcmp(dh1024_file, "")) {
     e = eventer_alloc();
     e->mask = EVENTER_ASYNCH;
     e->callback = generate_dh_params;
-    e->closure = (void *)512;
+    e->closure = (void *)1024;
     eventer_add_asynch(NULL, e);
   }
-  if (!dh1024_file || strcmp(dh1024_file, "")) {
+  if (!dh2048_file || strcmp(dh2048_file, "")) {
```

Comment:
2048-bit works just like 1024-bit does today.  We've not had a way to completely disable generating a given set of DH params, AFAIK.